R/geom-linerange.r

```diff
@@ -99,7 +99,7 @@ GeomLinerange <- ggproto("GeomLinerange", Geom,
 
   setup_params = function(data, params) {
     params$flipped_aes <- has_flipped_aes(data, params, range_is_orthogonal = TRUE)
-    if (!(params$flipped_aes || all(c("x", "ymin", "ymax") %in% names(data)))) {
+    if (!(params$flipped_aes || all(c("x", "ymin", "ymax") %in% c(names(data), names(params))))) {
```

Comment:
Just wondering why it tests for `x`, `ymin`, `ymax` but not `y`, `xmin`, `xmax` in `params`. Is this correct?

Reply:
if `has_flipped_aes()` returns true it means that it contains y, xmin, and xmax

Reply:
Ah, Ok, maybe add a quick comment.